provider/pkg/schema/gen.go

```diff
@@ -916,6 +960,117 @@ func (ctx *cfSchemaContext) propertySpec(propName, resourceTypeName string, spec
 	return &propertySpec, nil
 }
 
+// updateDesc updates the schema with a description from the CloudFormation docs if one
+// is not already provided
+func (ctx *cfSchemaContext) updateDesc(refName, propName string, spec *jsschema.Schema) {
```

Comment:
what if the existing docs are worse than the new docs? I wonder if we need some way to select between the two eventually in the future

Reply:
Yeah that's a good idea, i'll create an issue to track it.